react-tutorial/src/components/board/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./Board";
```

Comment:
You don't need two files (one for the export and one for the component), please delete this one and rename Board.js to index.js